tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MicrosoftDataSqliteTests.cs

```diff
@@ -50,7 +52,7 @@ public void SubmitsTraces(string packageVersion)
 
             foreach (var span in spans)
             {
-                var result = span.IsSqlite();
+                var result = ValidateIntegrationSpan(span);
                 Assert.True(result.Success, result.ToString());
 
                 Assert.Equal(expectedServiceName, span.Service);
```

Comment:
AFAICT, every test calls this, maybe it's worth extracting this whole snippet to a helper method on `TracingIntegrationTest`? e.g.

```csharp
public void ValidateIntegrationSpans(IEnumerable<MockSpan> spans, string expectedServiceName)
{
    foreach (var span in spans)
    {
        var result = span.IsSqlite();
        var result = ValidateIntegrationSpan(span);
        Assert.True(result.Success, result.ToString());

        Assert.Equal(expectedServiceName, span.Service);
        Assert.False(span.Tags?.ContainsKey(Tags.Version), "External service span should not have service version tag.");
    }
}
```

Reply:
Good idea! I've added the new method almost exactly as you suggested, and refactored as many tests as I could to use the new method in commit 748f445d1ffe9b0b1a740c2ecc6369aa18c3d4e3